modules/account-lib/src/coin/dot/iface.ts

```diff
@@ -170,3 +169,14 @@ export type FeeOptions = {
   formula?: 'fixed' | 'feeRate' | 'perKB' | 'custom';
   type?: 'base' | 'max' | 'tip';
 };
+
+export interface TransactionExplanation extends Omit<BaseTransactionExplanation, 'id'> {
```

Comment:
Does `Omit` mean we exclude the `id` field? That is the tx id, right?

Reply:
Yep

Reply:
But we should provide that, right? Or why do we omit it? Or is it optional?

Reply:
Dot doesn’t have ids. Some coins do, hence its in the base class.

Reply:
curious about lack of ID and how it'll impact our workflows.

In Solana, transaction id === transaction's first signature.
Which is only available for signed transactions.

I wonder how our system will handle transactions without IDs.